examples/with-relay-hooks/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "yarn run relay && next",
+    "build": "yarn run relay && next build",
+    "start": "yarn run relay && next start",
+    "relay": "yarn run relay-compiler $@",
+    "schema": "export $(grep -v '^#' .env | xargs) && get-graphql-schema $NEXT_PUBLIC_RELAY_ENDPOINT > schema.graphql"
+  },
+  "relay": {
+    "src": "./",
+    "schema": "./schema.graphql"
+  },
+  "dependencies": {
+    "next": "latest",
+    "react": "^18.0.0-rc.0",
+    "react-dom": "^18.0.0-rc.0",
+    "relay-runtime": "^13.0.2",
+    "react-relay": "^13.0.2"
+  },
+  "devDependencies": {
+    "babel-plugin-relay": "^13.0.2",
```

Comment:
Now we can also use first-class support for Relay in Next.js:
https://github.com/vercel/next.js/pull/33702

Reply:
I've updated this PR to use Next.js's built-in relay support 🚀